src/worker/index.ts

```
@@ -0,0 +1,308 @@
+          // TODO: Do we need any options here?
+          // useNewUrlParser: true,
+          // useUnifiedTopology: true,
+        }),
+        logger: this.#logger,
+      });
+
+    // TODO: Start processing
+    this.#delay(this.#queueInterval).then(() => this.#processMessages());
+    this.#delay(this.#queueInterval).then(() => this.#enqueueDelayedMessages());
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)